src/webapp/js/components/modules-form.js

```diff
@@ -353,19 +355,22 @@ function validateFields(fields, values) {
   const errors = {};
 
   _.each(fields, (fieldConfig, fieldName) => {
+    const fieldValue = values[fieldName];
+
     if (fieldConfig.type === 'section') {
-      errors[fieldName] = validateFields(fieldConfig.fields, values[fieldName]);
+      errors[fieldName] = validateFields(fieldConfig.fields, fieldValue);
     } else if (fieldConfig.type === 'array') {
-      const array = values[fieldName];
-
-      if (array) {
+      if (fieldValue) {
         errors[fieldName] = [];
-        _.each(array, (value) => {
+        _.each(fieldValue, (value) => {
           errors[fieldName].push(validateFields(fieldConfig.fields, value));
         });
       }
-    } else if (fieldConfig.required && (values[fieldName] === undefined || values[fieldName] === null || values[fieldName] === '')) {
+    } else if (fieldConfig.required && (fieldValue === undefined || fieldValue === null || fieldValue === '')) {
       errors[fieldName] = 'This field is required';
+    } else if (fieldConfig.minVal !== null && fieldConfig.minVal !== undefined
```

Comment:
Same case as above:
if ( !!fieldConfig.minVal && !!fieldValue && fieldValue < fieldConfig.minVal )

Reply:
as above

Reply:
true, then it is ok.